templates/General/uperf-v2.jsonnet

```diff
@@ -0,0 +1,32 @@
+local panels = import '../../assets/uperf/panels.libsonnet';
+local queries = import '../../assets/uperf/queries.libsonnet';
+local variables = import '../../assets/uperf/variables.libsonnet';
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+
+g.dashboard.new('Public - UPerf Results dashboard')
++ g.dashboard.withTags(['network', 'performance'])
++ g.dashboard.time.withFrom('now/y')
```

Comment:
This seems to incorrect according to [here](https://github.com/cloud-bulldozer/performance-dashboards/blob/master/templates/General/uperf-perf.jsonnet#L235).

Reply:
yes, my bad i will correct it